src/cider/nrepl/middleware/complete.clj

```diff
@@ -57,7 +60,8 @@
      :requires {"symbol" "The symbol to lookup"
                 "ns" "The symbol's namespace"
                 "session" "The current session"}
-     :optional {"context" "Completion context for compliment."}
+     :optional {"context" "Completion context for compliment."
+                "extra-metadata" "List of extra-metadata fields"}
```

Comment:
Maybe we should document the possible values `extra-metadata` can contain here too. AFAICT it's just `:arglists` and `:doc` for now?